.github/workflows/label-analysis.yml

```diff
@@ -46,6 +46,9 @@ jobs:
           cache-dependency-path: '**/requirements*.txt'
       - name: Install dependencies
         run: pip install -r requirements.txt -r tasks/requirements.txt
+      - name: Remove unsupported git config
+        # Triggers dulwich.repo.UnsupportedExtension: b'worktreeConfig' 
+        run: git config --unset-all extensions.worktreeconfig
```

Comment:
Can you link the upstream issue in code, so we don't forget: https://github.com/GitTools/actions/issues/1115?